README_中文_README_CHN

```diff
@@ -0,0 +1,186 @@
+# Euler-sdk
+
+A basic SDK for interacting with [Euler smart contracts](https://github.com/euler-xyz/euler-contracts).
+Currently this is an alpha software, under intensive development.
+
+用于与 [Euler 智能合约](https://github.com/euler-xyz/euler-contracts) 交互的基本 SDK。
+目前这是一个 alpha版本，正在密集开发中。
+
+## 安装 (Installation)
+```bash
+npm i @eulerxyz/euler-sdk
+```
+
+## Euler类 (Euler class)
+The package provides a single named export - the Euler class  
+本包提供了一个单独的命名导出 - Euler 类
+
+```js
+import { Euler } from "@eulerxyz/euler-sdk"
+```
+
+The constructor takes a single required arguement - `signerOrProvider`, similar to ethers. The second argument is the `chainId`. Currently the SDK contains built in configurations for mainnet (chainId = 1), which is also default, and ropsten (chainId = 3) deployments.
+
+构造函数接受一个必需的参数 - `signerOrProvider`，类似于 ethers。第二个参数是 `chainId`。目前 SDK 包含主网 (chainId = 1) 的内置配置，这也是默认配置，同时还有 ropsten (chainId = 3) 部署。
+
+```js
+const provider = new ethers.providers.JsonRpcProvider("<JSON_RPC_URL>")
+const signer = new ethers.Wallet("<PRV_KEY>", provider)
+
+const e = new Euler(signer);
+```
+
+## 与智能合约交互 (Interacting with the smart contracts)
+
+### 主要模块 (Main modules)
+By default, the SDK provides instances of the [ethers.Contract](https://docs.ethers.io/v5/api/contract/contract/) of the main Euler modules: Euler, Exec, Liquidation, Markets and Swap.
+
+默认情况下，SDK 提供主要 Euler 模块的 [ethers.Contract](https://docs.ethers.io/v5/api/contract/contract/) 实例：Euler、Exec、Liquidation、Markets 和 Swap。
+
+```js
+// activate a new market
+await e.contracts.markets.activateMarket(tokenAddress)
+
+// check to see if liquidation would be profitable
+const liquidationOpportunity = await e.contracts.liquidation.callStatic.checkLiquidation(liquidator, violator, underlying, collateral)
+```
+
+### 次要部分 (Peripheries)
+
+In addition to the main modules, the SDK contains configuration for the mining contracts EulStakes and EulDistributor as well as peripheries FlashLoan and EulerGeneralView. They need to be explicitly initiated:
+
+除主要模块外，SDK 还包含挖矿合约 EulStakes 和 EulDistributor 以及次要部分的 FlashLoan 和 EulerGeneralView 配置。它们需要被激活：
+
+```js
+e.addContract("EulStakes")
+e.addContract("FlashLoan")
+```
+
+### Eul代币 (Eul token)
+
+Euler's native governance token Eul is also available:  
+Euler 的原生治理代币 Eul 也可用：
+
+```js
+const balance = await e.contracts.eul.balanceOf(myAccount)
+```
+
+Additionally, the token configuration like decimals, logo, permit data are provided in `eulTokenConfig` property:
+
+此外，在 `eulTokenConfig` 属性中提供了小数、logo、许可数据等代币配置：
+
+```js
+const { logo, extensions: { permit: { domain } } } = e.eulerTokenConfig
+```
+
+### eTokens, dTokens, pTokens
+Contracts that exist for every activated market can be instantiated and accessed by helper methods `eToken(address)`, `dToken(address)`, `pToken(address)`
+
+每个激活市场的合约都可以通过辅助方法 `eToken(address)`、`dToken(address)`、`pToken(address)` 实例化和访问
```

Comment:
Thanks for submitting the PR! I'll be happy to merge it in. 

Just verifying the translations using deepl. Everything checks out minus this sentence, which gets translated back to english as:
`Each contract that activates the marketplace can be instantiated and accessed via the helper methods`
Maybe its an issue with the automatic translation, but just to make sure: eToken, dTokens and pTokens are created, by euler Markets contract when a new market is activated, not the other way round (they are not activating the market). Please confirm the translation is correct.

Reply:
i view `eToken(address)`、`dToken(address)`、`pToken(address)`  as sort of parameter/variable groups（as whole)  .Also words like "token" "address" are easy ones for non-English speaking coders to recognize.So it's better to keep original contexts in this specific sanario.Therefore i keep them all original.It's not a mistake

Reply:
Ok, thanks